types/codemirror/index.d.ts

```diff
@@ -1650,6 +1650,12 @@ declare namespace CodeMirror {
          * Get the string between the start of the current token and the current stream position.
          */
         current(): string;
+
+        /**
+         * Returns the content of the line n lines ahead in the stream without
+         * advancing it. Will return undefined if not found.
```

Comment:
According to the comment, the return type should be `string | undefined`.

Reply:
According to the code you linked, lookAhead takes four parameters `(str, consume, skipSpaces, caseInsensitive)` and returns a boolean. It also changes the state of the StringStream. Am I looking at the wrong code?

Reply:
Yeah I think the link is wrong 😳  The link references to v1 of CodeMirror

This is the current implementation: https://github.com/codemirror/CodeMirror/blob/bd1b7d2976d768ae4e3b8cf209ec59ad73c0305a/src/util/StringStream.js#L80

That `lineOracle.lookAhead` could be https://github.com/codemirror/CodeMirror/blob/bd1b7d2976d768ae4e3b8cf209ec59ad73c0305a/src/line/highlight.js#L25 which seems to return whatever `getLine` returns, which is the result of `chunk.lines[n]` meaning it can be either a `string` or `undefined` if I'm following the code correctly 🤔

Reply:
So I guess the only change needed is to change the return type to `string | undefined`